packages/mobile/src/identity/verification.ts

```diff
@@ -73,6 +74,7 @@ export function* waitForUserVerified() {
 }
 
 export function* startVerification() {
+  yield put(startVerificationState())
   yield put(resetVerification())
```

Comment:
Reset verification should probably reset the start state too.
How about instead of a new action, just have a true/false param to resetVerification that will also set the `startedVerification` state. The goal is to reduce the number of yield calls, which I've noticed to slow things down.